test/plots/piePlotTests.ts

```diff
@@ -464,6 +464,17 @@ describe("Plots", () => {
         assert.lengthOf(entitiesOutsideOuterRadius, 0, "no entities returned when clicking outside outerRadius()");
         svg.remove();
       });
+
+      it("returns undefined for entitiesNearest when no entities rendered", () => {
```

Comment:
Can we use a consistent name for the tests? `"retrieves undefined from entityNearest when no entities are rendered"` seems best.